### PR TITLE
feat(icons): Add new icons shift-up and shift-down

### PR DESCRIPTION
**Pull request by jialin-he**
**This change:** (check at least one)

- [x] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [x] Yes
- [ ] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [x] Changeset added?

**What does this change address?**
We need a vertical version of collapse and expand icon.

**How does this change work?**
Add the icons to pharos.

### CODE REVIEW

**Review by eslawski — APPROVED** [review]

**Review by satya-achanta-venkata — APPROVED** [review]

File: packages/pharos/tokens/assets/icons.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
       "cite": { "value": "assets/icons/cite.svg" },
       "close-inverse": { "value": "assets/icons/close-inverse.svg" },
       "close": { "value": "assets/icons/close.svg" },
-      "collapse": { "value": "assets/icons/collapse.svg" },
```
Comment (daneah) [inline packages/pharos/tokens/assets/icons.json:28]: If we're removing an icon and renaming another icon, these are both breaking changes.

Comment (daneah) [reply packages/pharos/tokens/assets/icons.json:28]: You can consider deprecating the existing icons for now, and they can be removed in a later major version release. We don't have a convention for deprecating tokens at the moment, but Style Dictionary offers a reasonable suggestion [here](https://github.com/amzn/style-dictionary/blob/main/examples/advanced/tokens-deprecation/tokens/color/base.json#L9-L13) that would look like this:

```json
"collapse": {
    "value": "assets/icons/collapse.svg",
    "deprecated": true,
    "deprecated_comment": "Use collapse-<direction> instead"
}
```

File: packages/pharos/tokens/assets/icons.json [hunk 2]
```diff
@@ -36,7 +35,10 @@
       "email": { "value": "assets/icons/email.svg" },
       "exclamation-inverse": { "value": "assets/icons/exclamation-inverse.svg" },
       "exclamation": { "value": "assets/icons/exclamation.svg" },
-      "expand": { "value": "assets/icons/expand.svg" },
```
Comment (daneah) [inline packages/pharos/tokens/assets/icons.json:39]: If we're removing an icon and renaming another icon, these are both breaking changes. You can consider deprecating the existing icons for now, and they can be removed in a later major version release. We don't have a convention for deprecating tokens at the moment, but Style Dictionary offers a reasonable suggestion [here](https://github.com/amzn/style-dictionary/blob/main/examples/advanced/tokens-deprecation/tokens/color/base.json#L9-L13) that would look like this:

```json
"collapse": {
    "value": "assets/icons/collapse.svg",
    "deprecated": true,
    "deprecated_comment": "Use collapse-<direction> instead"
}
```

**Review by daneah — CHANGES_REQUESTED** [review]

File: packages/pharos/tokens/assets/icons.json [hunk 3]
```diff
@@ -25,7 +25,11 @@
       "cite": { "value": "assets/icons/cite.svg" },
       "close-inverse": { "value": "assets/icons/close-inverse.svg" },
       "close": { "value": "assets/icons/close.svg" },
-      "collapse": { "value": "assets/icons/collapse.svg" },
+      "collapse": {
+        "value": "assets/icons/collapse.svg",
```
Comment (daneah) [inline packages/pharos/tokens/assets/icons.json:29]:
```suggestion
        "value": "assets/icons/shift-right.svg",
```

**Review by daneah — APPROVED** [review]

**Pull request merged**